linera-views/src/backends/scylla_db.rs

```diff
@@ -242,8 +281,11 @@ impl ScyllaDbClient {
         &self,
         root_key: &[u8],
         key: Vec<u8>,
-    ) -> Result<bool, ScyllaDbStoreError> {
-        ensure!(key.len() <= MAX_KEY_SIZE, ScyllaDbStoreError::KeyTooLong);
+    ) -> Result<bool, ScyllaDbStoreInternalError> {
+        ensure!(
+            key.len() <= MAX_KEY_SIZE,
+            ScyllaDbStoreInternalError::KeyTooLong
+        );
```

Comment:
We could extract and deduplicate this:

```suggestion
        self.check_key_size(&key)?;
```

Reply:
Great idea (and done for DynamoDb/RocksDb).